mobile/android/android-application-manager.ts

```diff
@@ -27,6 +29,16 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 		}).future<string[]>()();
 	}
 
+	public getApplicationsLiveSyncSupportedStatus(newApplications: string[]): IFuture<void> {
+		return (() => {
+			let liveSyncStatus = getFuturesResults<boolean>(_.map(newApplications, app => this.isLiveSyncSupportedForApplication(app)), () => true);
```

Comment:
Parallel or not you seem to issue one broadcast for every app which may lead to complications with more devices, more apps, etc. 
There may be a way of getting all applications to provide an answer with a single broadcast - perhaps by removing the `app-id` parameter of the broadcast :thought_balloon: